test/index.test.js

```diff
@@ -22,11 +22,15 @@ class FakeConfigData {
 }
 
 class FakeCommand {
-  constructor(args, config, secureStorage) {
+  constructor(args, config) {
     this.args = args;
     this.config = config;
-    this.secureStorage = secureStorage;
     this.ran = false;
+    this.userConfig = {
+      getProfileById() {
+        return false;
```

Comment:
[Q] : Why are we returning `false` always here ?

Reply:
We are mocking the method getProfileById, instead of secureStorage, to mock the current Profile being used. For mocking to work sinon requires the method to always be present. So added the method in FakeCommand. This method would never return false since it will always be mocked at line: 26. So its just a placeholder method.